client/src/components/FlexTable.js

```diff
@@ -0,0 +1,75 @@
+import React from "react";
+import "./FlexTable.css";
+import Icon from "./Icon";
+
+const FlexTable = ({ titleData = [], data = [], iconClick, tableId }) => {
+  return (
+    <div
+      id={tableId}
+      className="flexTable flexTable--5cols flexTable--collapse"
+    >
+      <div className="flexTable-row flexTable-row--head">
+        <div className="flexTable-row flexTable-row--head">
+          {titleData.map((item, k) => (
+            <div
+              key={k}
+              className={`flexTable-cell ${item.fieldName}-cell column-heading`}
+            >
+              {" "}
+              {item.title}
+            </div>
+          ))}
+        </div>
+      </div>
+      {data.map((item, i) => (
+        <React.Fragment key={"row" + i}>
```

Comment:
I think this React.Fragment is not necessary.

Reply:
Yes, certainly.